packages/admin/src/components/article/ArticleSearchSideBar.tsx

```
@@ -15,13 +43,16 @@ export const ArticleSearchSideBar = () => {
         <SearchInput inputText="기사 검색.." />
       </UpperWrapper>
       <ArticleWrapper>
+         {datas.map((data) => (
+          <ArticleDiv title={data.title} date={data.date} onClick={(id) => articleDivClick(data.id)}/>
+         ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)